src/script/components/InputBar/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -71,7 +71,7 @@ interface RichTextEditorProps {
   showFormatToolbar: boolean;
   showMarkdownPreview: boolean;
   getMentionCandidates: (search?: string | null) => User[];
-  saveDraftState: (editor: string) => void;
+  saveDraftState: (editor: string, plainMessage: string) => void;
```

Comment:
What about passing editorState as is, without stringifing it? The `saveToSimpleStorage` should stringify it automatically + can we make it an object?

```suggestion
  saveDraft: ({editorState, plainMessage}: {editorState: EditorState, plainMessage: string}) => void;
```

Reply:
I'll keep it like we have currently implemented for now :)